e2e/playwright/conftest.py

```diff
@@ -0,0 +1,419 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Global pytest fixtures for e2e tests.
+This file is automatically run by pytest before tests are executed.
+"""
+from __future__ import annotations
+
+import contextlib
+import os
+import re
+import shlex
+import shutil
+import socket
+import subprocess
+import sys
+import time
+from io import BytesIO
+from pathlib import Path
+from tempfile import TemporaryFile
+from types import ModuleType
+from typing import Any, Generator, List, Literal, Protocol
+
+import pytest
+import requests
+from PIL import Image
+from playwright.sync_api import ElementHandle, Locator, Page
+from pytest import FixtureRequest
+
+
+class AsyncSubprocess:
+    """A context manager. Wraps subprocess.Popen to capture output safely."""
```

Comment:
`"""A context manager. Wraps subprocess.Popen to capture output safely."""` Is this a typo?

Reply:
Fixed 👍